UsingRTT.md

```diff
@@ -287,26 +287,19 @@ debugger. Overhead for printf() calls included.
 
 ## Build instructions
 
-To compile with RTT support, add *ENABLE_RTT=1*.
+To compile with RTT support, add set the `rtt_support` option to true, this may be the default for some build configurations.
 
 Eg. for Black Magic Probe (native) hardware:
```

Comment:
```suggestion
For instance, for Black Magic Probe (native) hardware:
```

Reply:
We've applied this to the second instance of this under this one too.